### ISSUE TITLE
Ditch Disqus

### ISSUE THREAD

**Issue by Starz0r** [chocolatey/home#337]
### Checklist

- [x] I have verified this is the correct repository for opening this issue.
- [x] I have verified no other issues exist related to my request.

### Is Your Feature Request Related To A Problem? Please describe.

Disqus is one of those bad products that shouldn't need any introduction on why, but in case you're not in the loop, let me fill you in. Disqus is a service that provides a widget for commenting, that is now owned (well back in 2017 anyway) by a massive data driven, analytics, and marketing [conglomerate](https://en.wikipedia.org/wiki/Zeta_Global). In shorter terms: they make money by tracking and silently surveilling you from the small web-based JS polyfill they provide through Disqus. This of course, is no secret, and they have in fact, been doing this even before they were acquired.

Maybe, at the time, using this service was a good idea, but today, you're just allowing tons of users to be surveiled by a third-party without their consent. In a lot of ways, this is unacceptable, especially as this is entirely self-inflicted, and completely avoidable by just using another commenting system.

### Describe The Solution. Why is it needed?

While writing your own commenting system probably wouldn't be too difficult, I'm just going to recommend [Isso](https://isso-comments.de/). It's pretty well maintained, written in understandable Python and some C extensions. And even allows for importing existing Disqus comments!

### Additional Context

I understand the need for comments, but do not think it's worth using Disqus any longer. Especially since the longer it's kept around, the more information and data it will continue to gather on your users. As a Package Maintainer, I have Disqus completely blocked on my network, and have no desire to ever unblock or use it. This puts me in a situation where sometimes I'll get Emails about comments that I can't respond to because of this. I can't disable these Emails, or opt-out of this system, so I'd like to get to a place where this isn't a problem for me (and probably many others) anymore.

**Comment by pauby** [comment]
Duplicate of #70 

Was converted to discussion #162

**Comment by Starz0r** [comment]
Didn't even realize I wrote a similar issue back in 2021, dang.

**Comment by pauby** [comment]
At least you're consistent 😄
